### DISCUSSION TITLE
Rancher ingress needs port 8443 to be open

### DISCUSSION THREAD

**Discussion by jeroenhartgers** [rancher/rke2#2165] [Q&A] [answered]
After installing RKE2 on RHEL 8.4 on AWS EC2, I learnt the nasty way (well, took me 15 - 30 minutes to figure out) that the Rancher Ingress also uses port 8443 to communicate. Even though RKE2 is not Rancher, it is intended for Rancher.  I would still strongly advice to put port 8443 in the ports list table for RKE2, even if it is not mandatory for RKE2 itself. There seems to be no such table for Rancher, so IMO it seems fair. What do you think?

**Comment by brandond** [comment, marked as answer]
I don't believe there's any reason for us to add Rancher-specific ports to the RKE2 docs. The Rancher ports are documented here:
https://rancher.com/docs/rancher/v2.6/en/installation/requirements/ports/#ports-for-rancher-server-nodes-on-rke2

**Comment by jeroenhartgers** [comment]
Got it, thank you.  I got lost a little bit...
